src/arch/xtensa/configs/override/renoir_chrome_dts.config

```diff
@@ -1,2 +1,3 @@
+CONFIG_COMP_IIR=y
```

Comment:
COMP_IIR is included by default so adding this line should make no difference.
On the other hand, there is no harm by adding this for more assurance.